openflow13/match.go

```diff
@@ -1482,6 +1483,43 @@ func NewTunnelIpv6DstField(tunnelIpv6Dst net.IP, tunnelIpv6DstMask *net.IP) *Mat
 	return f
 }
 
+type TtlField struct {
+	Ttl uint8
+}
+
+func (m *TtlField) Len() uint16 {
+	return 1
+}
+
+func (m *TtlField) MarshalBinary() (data []byte, err error) {
+	data = make([]byte, 1)
+	data[0] = m.Ttl
+	return
+}
+
+func (m *TtlField) UnmarshalBinary(data []byte) error {
+	if(len(data) != 1) {
```

Comment:
Use `len(data) < m.Len()` instead, 1) better to not use the constant value directly for not easy to maintain if any change is introduced in the struct properties in future, 2) the data length is possibly larger than 1 if other fields existing in the bytes following the current one.

Reply:
Use the same error format as the errors in the existing code.

Reply:
parenthesis is not needed by convention

Reply:
Done. I  check the length of data as the existing code. 
 Can you help me review this patch again ? Thanks !
@tnqn  @wenyingd